ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -253,7 +253,7 @@ pattern IncorrectClaimedFlag claimedFlag <-
           (Alonzo.UtxosFailure
             (Alonzo.ValidationTagMismatch
               (Alonzo.IsValid claimedFlag
-      )))))
+      ) _))))
```

Comment:
Minor: this feels easy to overlook to me. A couple ideas:

- rename it `_<foo>` if only so it's easier to notice

- change the layout (perhaps of the whole pattern) so the `_` is on its own line